src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/ThrowHelper.cs

```diff
@@ -1,26 +1,155 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using Microsoft.Quic;
```

Comment:
Looks good, could you just move the file to System/Net/Quic/Internal so it doesn't wreak complete havoc on my PRs :smile: